app/models/form_element.rb

```diff
@@ -26,8 +28,10 @@ def set_position
   end
 
   def set_name
-    unless ActionKitFields::ACTIONKIT_FIELDS_WHITELIST.include?(name)
-      self.name = "action_#{name}" unless name =~  ActionKitFields::VALID_PREFIX_RE
+    unless name.blank? || ActionKitFields::ACTIONKIT_FIELDS_WHITELIST.include?(name)
```

Comment:
no, you can't save blank names. but this gets run before the validation, so the name is blank if the form is submitted with a blank name.